.github/ISSUE_TEMPLATE/route-shield-request.md

```diff
@@ -0,0 +1,26 @@
+---
+name: Route shield request
+about: Tell us about a route shield that's not on the map
+title: ''
+labels: shields
+assignees: ''
+
+---
+
+**Where can this route shield be found?**
+Example: Washington, DC, United States
+
+**Are there any `route=road` relations on OpenStreetMap corresponding to the route(s) where this shield is present?**
+If so, what are their `network=*` and `ref=*` values? If not, why not?
+Example: https://www.openstreetmap.org/relation/1858211, `network=US:DC`, `ref=295`
+
+**Describe the appearance of the shield.**
+Try to be specific with shapes, colors and placement so that the reader can mentally visualize the shield without looking at an image.
+Example: The route shield of Washington, DC is a white portrait-oriented rectangle with a black border. Inside the rectangle is a black outline of Washington, DC. The black text "DC" is in the top part of this outline, and the bottom part of the outline is broken to accommodate the route number in black.
+
+**Add a photo of the real-life shield.**
+Photos are preferred, but if they're unavailable, digital artwork is okay.
```

Comment:
This might be a good place to point to the part of the contributing guidelines that expresses a preference for guide sign–style shields over standalone markers.